### PR TITLE
Second Task

### PR DESCRIPTION
**Pull request by SiarheiPatapenka**
Hi Dzmitry , could you please have a look at my PR? Thank you!

### CODE REVIEW

**Comment by dshaplyko** [comment]
The task is accepted, but please fix the comments I placed in your free time. Thank you!

File: tasks/condition/condition.js [hunk 1]
```diff
@@ -7,11 +7,12 @@
   const a = 3,
         b = 2;
 
-  if (a + b < 4) {
-    console.log('wrong');
-  } else {
-    console.log('too much');
-  }
+  console.log(a+b<4 ? "wrong" : "too much")
```
[the remark this answers is not in the document]
Comment (SiarheiPatapenka) [reply tasks/condition/condition.js]: done

File: tasks/cycles/cycles.js [hunk 2]
```diff
@@ -20,28 +20,67 @@ function calculateDiscount(redemption) {
...
 }
 
 /**
  * return concatenated string from an array of substring
  */
 {
+  let sum='';
   const substr = ["I", " love", " JS"];
+  for (let element of substr) {
+    sum=substr.join('');
```
[the remark this answers is not in the document]
Comment (SiarheiPatapenka) [reply tasks/cycles/cycles.js]: done

File: tasks/cycles/cycles.js [hunk 3]
```diff
@@ -20,28 +20,67 @@ function calculateDiscount(redemption) {
...
-{
+{ let total=0;
   const personIncomes = {
     salary: 1985,
     rent: -600,
     interestOnDeposit: 250,
     otherExpences: -300
   };
+  for  (let property in personIncomes ) {
+    (total+=personIncomes[property]);
```
[the remark this answers is not in the document]
Comment (SiarheiPatapenka) [reply tasks/cycles/cycles.js]: done

**Pull request merged**